### DISCUSSION TITLE
라벨링 한 데이터를 삭제 해버렸습니다.

### DISCUSSION THREAD

**Discussion by po2955** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#40] [2024년도 기타 질문]
roboflow에서 301장을 라벨링 하고 Dataset에 올렸다가 200장 선택하고 삭제를 눌러버렸습니다.
301장 라벨링 한 기록이 있는데 과제 제출할 때 그 기록으로 제출해도 되겠습니까?

**Comment by Jinsun-Lee** [comment]
제출해도 됩니다.

**Reply by po2955** [reply]
감사합니다!
